phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexRebuildTaskIT.java

```diff
@@ -129,45 +128,42 @@ public void testIndexRebuildTask() throws Throwable {
             Task.addTask(conn.unwrap(PhoenixConnection.class), PTable.TaskType.INDEX_REBUILD,
                     TENANT1, null, viewName,
                     PTable.TaskStatus.CREATED.toString(), data, null, startTs, null, true);
-
-
+            conn.commit();
```

Comment:
Likewise I don't understand why we need the commit here (and below)

Reply:
Removed the commit after adding task. The connection is setAutocommit(false)